paper/paper.md

```diff
@@ -0,0 +1,281 @@
+  QSIPREP [@CieslakQSIPrep:2020], MRtrix3_connectome [@Smith2019:BIDSApp], NDMG [@KiarAVariability:2018]
+  and PyNets [@PisnerPyNets:2020], which have all demonstrated their capability in addressing all the previously-mentioned
+  challenges.
+However, none of the existing solutions provide a direct alternative to Connectome Mapper.
+
+Connectome Mapper (CMP) is an open-source pipeline software with a graphical user interface, designed
+  to simplify the organisation and the analysis of sMRI, dMRI, and rfMRI from raw data to multi-scale
+  structural weighted and functional connectomes [@Daducci:2012], using in a common framework
+  a multi-scale extension of the Desikan-Killiany parcellation [@Desikan2006AnInterest],
+  the so-called Lausanne brain parcellation [@Cammoun2012:MappingMRI], before the emergence of BIDS.
+While CMP3 derives from the two preceding versions and keeps the same philosophy, it has made CMP massively
+  evolve over the years in terms of the underlying codebase, the tools used, and the scope of the functionality
+  provided, including the migration to Python 3, a brand-new Lausanne parcellation scheme, the adoption
+  of the BIDS standard for data organization, the encapsulation of the processing pipelines in software
+  container images, continuously tested in concordance to the BIDS Apps standard, major upgrades of the diffusion and
+  fMRI pipelines, and the current extension to Electro-EncephaloGraphy (EEG), initiated during OHBM BrainHack 2020
+  ([https://github.com/ohbm/hackathon2020/issues/214](https://github.com/ohbm/hackathon2020/issues/214)).
+CMP3 has been designed around different characteristics and principles along which it is summarized
+  in this manuscript.
+
+# Summary
+
+\textbf{A flexible and interoperable workflow for multi-modal human connectome mapping.}
+Connectome Mapper 3 (CMP3) implements a workflow that creates a hierarchical multi-scale
+  connectome representation of the structural and functional brain systems, from any
+  sMRI / dMRI / rfMRI dataset structured according to the BIDS standard.
+It relies on Nipype [@GorgolewskiNipype:2011] and adopts a modular architecture.
+As illustrated by \autoref{fig:cmp3-diagram}, the workflow is composed of three different
+  pipelines (anatomical, diffusion, and fMRI) dedicated to the processing of each modality (sMRI, dMRI, rfMRI).
+
+![\textbf{Overview of the Connectome Mapper 3 BIDS App's workflow.}
+\label{fig:cmp3-diagram}](cmp3-diagram.png)
+
+Each pipeline is represented by a Nipype workflow that takes a BIDS formatted dataset as input, and
+  runs a number of sub-workflows (stages).
+Each stage implements one or multiple tasks, where each task can interface with either
+  a specific tool including in FSL [@Jenkinson2012FSL], FreeSurfer [@Fischl2012FreeSurfer],
+  ANTs [@AVANTS2008SymmetricBrain], dipy [@Garyfallidis2014DipyData],
+  mrtrix3 [@Tournier2019MRtrix3:Visualisation], AFNI [@Cox2012], or with an in-house tool
```

Comment:
unclear what is meant by inhouse tool: either cite or shortly describe what functions are included